packages/api/src/client/internal/generated/openapi/namespaces/Operations.ts

```diff
@@ -0,0 +1,31 @@
+import type { PreviewMode } from "../components/PreviewMode";
+import type { AsyncOperation } from "../components/AsyncOperation";
+import { OpenApiRequest } from "../request";
+
+/**
+ * Get an asynchronous operation by its ID.
+ *
+ * Third-party applications using this endpoint via OAuth2 must request the following operation scope: `api:read-data`.
+ *
+ */
+export function getOperation<TResponse>(_request: OpenApiRequest<AsyncOperation, TResponse>, operationId: string, preview?: PreviewMode): Promise<TResponse> {
+    return _request(
+        "getOperation",
+        "GET",
+        "/v2/operations/{operationId}",
+        __undefined,
+        __undefined,
+        {
+            preview,
```

Comment:
What is the rationale between taking `operationId: string, preview?: PreviewMode` vs taking `operationId: string, customHeaders?: { preview?: PreviewMode }`? Basically, why make things more complex than they need to be?